### DISCUSSION TITLE
Etar Calendar installation folder (Android)

### DISCUSSION THREAD

**Discussion by JohnnyBrav0Sr** [Etar-Group/Etar-Calendar#1650] [Q&A]
Hi

I have installed Etar Calendar from F-droid repository. I noticed on github it has a much newer version. I uninstalled it so I can install the github version. Now I receive an error when trying to install the github version, probably due to a different app signature.

I searched for a folder with the name 'Etar' in it in the Android/data folder, but with no success. Could you please tell me the exact name of the Etar Calendar installation folder, so I can whipe the remains of my Etar Calendar installation from F-droid? Thanks!

**Comment by jspricke** [comment]
We don't provide Github releases and the CI builds are not sigend so you probably so that as the error. There is also no "installations folder". The newer version are actually only build fixes for F-Droid that should be available from F-Droid soon.
